docs/data/product/dea-high-and-low-tide-imagery-landsat/_data.yaml

```
@@ -48,7 +48,7 @@ tags:
 # Access
 
 maps:
-  - link: https://maps.dea.ga.gov.au/#share=s-Vd0118nY3Dg0kUBNIi4a0y600r
+  - link: https://maps.dea.ga.gov.au/#share=s-1EbpIqbUuXAsK2qXNAjdr1SgS1w
```

Updated link to showcase both the low and high tide product layers:

```suggestion
  - link: https://maps.dea.ga.gov.au/#share=s-7YeWZKqWkF0Ctv5Av8RkHSWeNzQ
```

---

That's such a great idea!